lib/data/models/mostro_message.dart

```diff
@@ -118,6 +118,7 @@ class MostroMessage<T extends Payload> {
     required String recipientPubKey,
     NostrKeyPairs? masterKey,
     int? keyIndex,
+    int targetDifficulty = 16,
```

Comment:
Why default 16? probably that's not a good idea for some mobiles, in fact I don't think we need to add proof of work for chat between users, they are not going to spam each other

Reply:
Please, let's remove that proof of work logic from the chat